addons/mail/models/mail_channel.py

```diff
@@ -417,12 +419,13 @@ def _send_guidelines(self, partners):
         ]).mapped('email')
         for partner in partners.filtered(lambda p: p.email and not (p.email in banned_emails)):
             create_values = {
+                'email_from': partner.company_id.catchall_formatted or partner.company_id.email,
```

Comment:
Not using `email_formatted` here?

Also, I know it was already like this, but it's strange to fallback to something here and not above:
`'email_from': self.env.user.company_id.catchall_formatted,`

Reply:
Done